tests/test_allocations_specific_rules.py

```diff
@@ -333,10 +335,10 @@ def test_roc_strategy(df):
 
     assert pd.Series.equals(df_with_allocations["allocation"], df_with_signals["allocation"])
 
-
-@pytest.mark.parametrize("df", dataframes)
+'''@pytest.mark.parametrize("df", dataframes)
```

Comment:
This disables Vortex strategy's test completely. I think @NikolaR01 had a fix for the test in a PR branch?

Reply:
@ta-oliver @Optimus-Q We could merge into the infertrade_new_rules branch and resolve it there, possible conflicts after merging the "add_macd_adx" branch since they both add the MACDADX strategy

Reply:
Fixed the test for Vortex